docs/UserGuide.md

```diff
@@ -458,7 +461,7 @@ A quick overview of all supported commands, their formats and examples are given
 | Term                                         | Description                                               |
 | -------------------------------------------- | --------------------------------------------------------- |
 | **API** | <a name="glossary-api"></a> API is short for **Application Programming Interface** and allows two systems to interact with each other  |
-| **Call** | <a name="glossary-call"></a> A call to an API endpoint refers to the process of sending a [request to the server and then receiving a response](#83-what-are-requests-and-responses).          |
+| **Call** | <a name="glossary-call"></a> A call to an API endpoint refers to the process of sending a [request to the server and then receiving a response](#83-what-are-requests-and-responses)          |
```

Comment:
nice catch on fullstop

Reply:
This is the reason why we need all members to run through the UG once.